<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerfMonDialog</class>
 <widget class="QDialog" name="PerfMonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scash - Performance monitor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_network">
      <attribute name="title">
       <string>&amp;Network performance</string>
      </attribute>
      <layout class="QGridLayout" columnstretch="0">
       <property name="horizontalSpacing">
        <number>12</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Incoming traffic:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelIncomingTraffic">
           <property name="text">
            <string>                Loading...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Outgoing traffic:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelOutgoingTraffic">
           <property name="text">
            <string>                Loading...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Incoming connections count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelIncomingConnections">
           <property name="text">
            <string>                Loading...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Outgoing connections count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelOutgoingConnections">
           <property name="text">
            <string>                Loading...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Banned hosts:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelBannedCount">
           <property name="text">
            <string>                Loading...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_connections">
      <attribute name="title">
       <string>Network &amp;connections</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_local">
      <attribute name="title">
       <string>&amp;Local performance</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>3</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
